etna/whatson/migrations/0005_eventpage_need_to_know_button_link_and_more.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.2.7 on 2023-11-16 13:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("whatson", "0004_eventsession_session_id"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="eventpage",
+            name="need_to_know_button_link",
+            field=models.URLField(
+                blank=True,
+                help_text="The website for need to know info.",
+                max_length=255,
+                verbose_name="need to know link",
+            ),
+        ),
+        migrations.AddField(
+            model_name="eventpage",
+            name="need_to_know_button_text",
+            field=models.CharField(
+                blank=True,
+                help_text="The text of the need to know button.",
+                max_length=30,
+                verbose_name="need to know button text",
+            ),
+        ),
+        migrations.AddField(
+            model_name="eventpage",
+            name="venue_directions",
+            field=models.URLField(
+                blank=True,
+                help_text="A link to the venue’s ‘How to find us’ page.",
```

Comment:
```suggestion
                help_text="A link to the venue's 'How to find us' page.",
```

I'm not sure if we should use `‘`